### PR TITLE
Advance Rating

### PR DESCRIPTION
**Pull request by binh168**
==> Title: Statistical rating

==> Redmine ticket: https://edu-redmine.sun-asterisk.vn/projects/open-ruby-2019-hiring-player

==> Attachment:

### CODE REVIEW

**Comment by binh168** [comment]
ready

File: app/views/orders/_new.html.erb [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
+      </ul>
+    </div>
+    <div class="col-md-12">
+      <%= link_to(new_order_path(user_id: @user.id),
+       {data: {toggle: "modal", target: "#myModalBooking"}}, remote: true) do %>
+        <h4><button type="button" class="btn btn-primary"><%= t ".booking" %></button></h4>
+      <% end %>
+    </div>
+  </div>
 <% end %>
```
Comment (phamhuuluan) [inline app/views/orders/_new.html.erb]: thừa dòng này a

**Comment by binh168** [comment]
ready

**Comment by binh168** [comment]
ready

File: app/models/rating.rb [hunk 2]
```diff
@@ -1,5 +1,7 @@
 class Rating < ApplicationRecord
   has_one :sender_recipient, as: :send_recieve
 
+  scope :list_rating, -> (rating_id){select(:rating).where(id: rating_id)}
```
Comment (ngocvt-0484) [inline app/models/rating.rb]: đặt tên scope là by_ids đi e ơi

File: app/views/orders/_new.html.erb [hunk 3]
```diff
@@ -1,10 +1,17 @@
...
-  <% end %>
+  <div class="row">
+    <div class="col-md-12">
+      <ul class="rating-form">
+        <%= render "ratings/form" %>
+      </ul>
+    </div>
+    <div class="col-md-12">
+      <%= link_to(new_order_path(user_id: @user.id),
+       {data: {toggle: "modal", target: "#myModalBooking"}}, remote: true) do %>
```
Comment (ngocvt-0484) [inline app/views/orders/_new.html.erb]: ident

**Comment by binh168** [comment]
ready

**Pull request merged**